operators/src/processing/point_in_polygon/tester.rs

```diff
@@ -90,6 +99,39 @@ impl<'a> PointInPolygonTester<'a> {
         }
     }
 
+    fn precalculate_polygon_bounds(
```

Comment:
this should be easy to parallelize, right?

Reply:
yes but lets do parallelization as an extra task. It will change a lot of other things, too.